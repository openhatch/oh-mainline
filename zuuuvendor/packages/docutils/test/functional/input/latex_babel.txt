The babel_ package introduces the concept of "shorthands": additional
characters that introduce a latex macro. Most common is the active double
quote ("). Problematic is the tilde character (~) which is regularely used
for no-break spaces but redefined by some language definition files:

English: 'an' "active"-quote, ^circumflex, and no-break spaces

.. class:: language-eu

Basque: 'an' "active"-quote, ^circumflex, and no-break spaces

.. class:: language-eo

Esperanto: 'an' "active"-quote, ^circumflex, and no-break spaces

.. class:: language-et

Estonian: 'an' "active"-quote, ^circumflex, and no-break spaces

.. class:: language-gl

Galician: 'an' "active"-quote, ^circumflex, and no-break spaces

.. class:: language-de

German: 'an' "active"-quote, ^circumflex, and no-break spaces

Spanish: option clash with Galician!

..
   .. class:: language-es

   Spanish: 'an' "active"-quote, ^circumflex, and no-break spaces


.. _babel: http://www.ctan.org/packages/babel
