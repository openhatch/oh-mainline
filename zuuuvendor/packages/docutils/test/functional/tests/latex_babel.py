# Source and destination file names.
test_source = "latex_babel.txt"
test_destination = "latex_babel.tex"

# Keyword parameters passed to publish_file.
reader_name = "standalone"
parser_name = "rst"
writer_name = "latex"

# Extra setting we need
