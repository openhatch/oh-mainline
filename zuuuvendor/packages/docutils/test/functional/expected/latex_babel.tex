\documentclass[a4paper]{article}
% generated by Docutils <http://docutils.sourceforge.net/>
\usepackage{fixltx2e} % LaTeX patches, \textsubscript
\usepackage{cmap} % fix search and cut-and-paste in Acrobat
\usepackage{ifthen}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[basque,esperanto,estonian,galician,ngerman,english]{babel}
\AtBeginDocument{\shorthandoff{.<>}}
\deactivatetilden % restore ~ in Galician
\makeatletter
  \addto\extrasestonian{\bbl@deactivate{~}}
\makeatother
\makeatletter
  \addto\extrasbasque{\bbl@deactivate{~}}
\makeatother

%%% Custom LaTeX preamble
% PDF Standard Fonts
\usepackage{mathptmx} % Times
\usepackage[scaled=.90]{helvet}
\usepackage{courier}

%%% User specified packages and stylesheets

%%% Fallback definitions for Docutils-specific commands

% hyperlinks:
\ifthenelse{\isundefined{\hypersetup}}{
  \usepackage[colorlinks=true,linkcolor=blue,urlcolor=blue]{hyperref}
  \urlstyle{same} % normal text font (alternatives: tt, rm, sf)
}{}


%%% Body
\begin{document}

The \href{http://www.ctan.org/packages/babel}{babel} package introduces the concept of \textquotedbl{}shorthands\textquotedbl{}: additional
characters that introduce a latex macro. Most common is the active double
quote (\textquotedbl{}). Problematic is the tilde character (\textasciitilde{}) which is regularely used
for no-break spaces but redefined by some language definition files:

English: 'an' \textquotedbl{}active\textquotedbl{}-quote, \textasciicircum{}circumflex, and~no-break~spaces

\foreignlanguage{basque}{Basque: 'an' \textquotedbl{}active\textquotedbl{}-quote, \textasciicircum{}circumflex, and~no-break~spaces}

\foreignlanguage{esperanto}{Esperanto: 'an' \textquotedbl{}active\textquotedbl{}-quote, \textasciicircum{}circumflex, and~no-break~spaces}

\foreignlanguage{estonian}{Estonian: 'an' \textquotedbl{}active\textquotedbl{}-quote, \textasciicircum{}circumflex, and~no-break~spaces}

\foreignlanguage{galician}{Galician: 'an' \textquotedbl{}active\textquotedbl{}-quote, \textasciicircum{}circumflex, and~no-break~spaces}

\foreignlanguage{ngerman}{German: 'an' \textquotedbl{}active\textquotedbl{}-quote, \textasciicircum{}circumflex, and~no-break~spaces}

Spanish: option clash with Galician!

% .. class:: language-es
% 
% Spanish: 'an' "active"-quote, ^circumflex, and no-break spaces

\end{document}
